app/transformation/morph-enriched.xml

```diff
@@ -17,8 +17,13 @@
 						<regexp match="(www(.*))" format="http://${1}"/>
 						<replace pattern="\s+" with=""/>
 					</data>
-					<combine value="${hp}" name="$[group_name]" sameEntity="true">
-						<choose>
+					<combine value="${hp}" name="$[group_name]" sameEntity="true" flushWith="009Q.z" reset="true">
+						<if>
+							<data source="009Q.z" name="">
```

Comment:
No, it's optional, see https://github.com/hbz/metafacture-core/blob/master/src/main/resources/schemata/metamorph.xsd#L472